src/main/java/mediathek/tool/ApplicationConfiguration.java

```diff
@@ -90,7 +90,7 @@
 
     public static final String FILM_SHOW_DESCRIPTION = "film.show_description";
 
-    private static ApplicationConfiguration ourInstance;
+    private static final ApplicationConfiguration ourInstance = new ApplicationConfiguration();
```

Comment:
Verschlechtert die Performance beim start.

Reply:
Es handelt sich um einen Rückbau der Änderung in 7b5aca5ed4a378fbd53280677c5c9cc7c0248b53
(vgl. https://forum.mediathekview.de/topic/2827/exception-bei-aktueller-nightly-ohne-bestehenden-config-ordner/7)
auf den alten Stand, der in master vorhanden ist.

Reply:
Das lazy loading, eingeführt in SHA-1: 7b5aca5ed4a378fbd53280677c5c9cc7c0248b53 kam als Versuch der Behebung der gemeldeten NullPointerException. Durch dieses lazy loading aber kam es (durch den Rekursiven Aufruf des Konstruktors durch den Konstruktor beim Allokieren der Singleton-Instanz) aber zu einem StackOverflow.

Ich kenne den MV Quellcode kaum und es handelt sich hier lediglich um einen Vorschlag um die gemeldete Nullpointer-Exception herum. sh. https://forum.mediathekview.de/topic/2827/exception-bei-aktueller-nightly-ohne-bestehenden-config-ordner/9

Ich kann nochmal nachdenken, ob ich Ideen habe. ohne Gewähr.